src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -275,10 +275,14 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
   def registerUserInRawls(
     pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
-    requestContext: RequestContext): Future[PerRequestMessage] = {
+    requestContext: RequestContext, userId: String): Future[PerRequestMessage] = {
     pipeline { Post(UserService.rawlsRegisterUserURL) } map { response =>
         response.status match {
           case x if x.isSuccess || isConflict(response) =>
+            //sendNotification is purposely detached from the sequence. we don't want to block
+            //the completion of registration by waiting for sendgrid to succeed or fail the registration
+            // if sendgrid fails. unfortunately, if this call fails then ¯\_(ツ)_/¯
+            sendNotification(pipeline, ActivationNotification(userId))
```

Comment:
I agree, it is an unrecoverable error from the client's perspective and should be logged, not passed through.